src/main/java/use_case/follow/FollowUserDataAccessInterface.java

```diff
@@ -0,0 +1,9 @@
+package use_case.follow;
+
+import entity.User;
+
+public interface FollowUserDataAccessInterface {
```

Comment:
same comment that was left in the data access object, i don't actually think you need these data access interfaces and can instead use the one given by TierListDataAccessInterface or just copy paste the same method over